bvm/ballerina-runtime/README.md

```diff
@@ -0,0 +1,256 @@
+| stream             | io.ballerina.runtime.api.values.BStream          |                                                                        |
+| object             | io.ballerina.runtime.api.values.BObject          |                                                                        |
+| future             | io.ballerina.runtime.api.values.BFuture          |                                                                        |
+| function           | io.ballerina.runtime.api.values.BFunctionPointer |                                                                        |
+| typedesc           | io.ballerina.runtime.api.values.BTypedesc        |                                                                        |
+| error              | io.ballerina.runtime.api.values.BError           |                                                                        |
+
+## Jballerina main API constructs
+
+|                                          |                                                                                                                                                                                                                                                                                                                                                                             |
+|------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| **Construct**                            | **Description**                                                                                                                                                                                                                                                                                                                                                             |
+| io.ballerina.runtime.api.Environment     | Developers can use this as the first argument of an interop method, Ballerina will inject an instance of Environment when calling. That instance can be used to communicate with the currently executing Ballerina runtime.With Environment you can get interop ballerina function name, path parameters, strand id, strand metadata, current module , current runtime etc. |
+| io.ballerina.runtime.api.Future          | This will contain the future value once we call the ballerina method from API asynchrounously.                                                                                                                                                                                                                                                                              |
```

Comment:
```suggestion
| io.ballerina.runtime.api.Future          | This will contain the future value once we call the ballerina method from API asynchronously.                                                                                                                                                                                                                                                                              |
```